include/fastdds/dds/domain/DomainParticipantFactory.hpp

```diff
@@ -242,6 +242,12 @@ class DomainParticipantFactory
     void participant_has_been_deleted(
             DomainParticipantImpl* part);
 
+    /**
+     * Load statistics profiles from environment variable
+     * @return string with the semicolon-separated list of statistics topic names to be enabled
+     */
+    const std::string& load_statistics_profiles_from_env();
```

Comment:
I think this should be on a private header / source. As it is only going to be used on the creation of a participant, I propose to add it on `DomainParticipantFactory.cpp`. It could then be a file static method.